server/bin/pbench-server-prep-shim-002.sh

```diff
@@ -187,6 +187,13 @@ while read tbmd5 ;do
         (( nerrs++ ))
         continue
     fi
+    # mv does not restore the SELinux context properly, so we do it by hand
+    restorecon ${dest}/${tb}
```

Comment:
Don't we also have to issue a `restorecon` when the `${dest}` is created?  Perhaps we should just issue one `restorecon -R ${dest}`

Reply:
The `mkdir` creating the directory does set the context properly (assuming that the parent directory is set correctly which the installation playbook takes care of). So yes, we can, but I don't think it's buying us anything.

Reply:
If you are confident that you the directory is all set, then let's go with it as it is.  We just need to fix the unit tests, which failed in TravisCI for some odd reason.

Reply:
Yes, I'm pretty sure it's OK. I undid the test-29.0 change which is what failed in travis (of course, it's now failing on my machine - sigh...)